pom.xml

```
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.5.4</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.example</groupId>
+	<artifactId>user-manage-apirest</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>user-manage-apirest</name>
+	<description>Manage user</description>
+	<properties>
+		<java.version>11</java.version>
+	</properties>
```

Las versiones deben ir en properties

```suggestion
	<properties>
		<java.version>11</java.version>
		<org.springdoc.version>1.5.10</org.springdoc.version>
	</properties>
```